### DISCUSSION TITLE
Error when attempting to add a description to a device interface

### DISCUSSION THREAD

**Discussion by jeyriku** [netbox-community/netbox#10454] [Q&A] [answered]
Hi All,

I'm requesting your help because i'm facing an issue when trying to add a description to a device interface.

<class 'django.utils.datastructures.MultiValueDictKeyError'>

'device'

Python version: 3.8.10
NetBox version: 3.3.3

Let me know if you need some more details.

Thanks in advance for your kind help !

**Comment by candlerb** [comment, marked as answer]
See the [release notes for Netbox v3.3.4](https://github.com/netbox-community/netbox/releases/tag/v3.3.4):

> #10387 - Fix MultiValueDictKeyError exception when editing a device interface

**Reply by jeyriku** [reply]
Thank you so much @candlerb
